### DISCUSSION TITLE
Request/Response Caching

### DISCUSSION THREAD

**Discussion by lukaszflorczak** [nuxt-modules/algolia#111] [Q&A] [answered]
Hi,
How to use request/response caching (https://www.algolia.com/doc/api-client/getting-started/customize/javascript/?client=javascript#caching-requests-and-responses) with the module?

**Comment by Baroshem** [comment, marked as answer]
Hey,

The module does not support this feature yet as it seems to be quite a new one. You can recommend it as an issue and I can work on it as a part of the next release.

I believe that it only requires an options object to be passed to the useAlgoliaSearch and useAlgoliaRef composables and maybe bump some packages.

Or, you can always create a helper function that will store the result of search in memory cache. It will work very similar to the approach shown in the docs. It can be implemented by using useStorage from Nuxt or by package 'memory-cache'.

Let me know what solution would work for you best :)

**Reply by lukaszflorczak** [reply]
Because Algolia has this feature probable it could be better to use it and don't duplicate it with a custom solution. I'll open an issue. 

Thank you for your answer!
